support/config/deployment.go

```diff
@@ -269,6 +269,8 @@ func (c *DeploymentConfig) setControlPlaneIsolation(hcp *hyperv1.HostedControlPl
 		})
 	}
 
+	c.Scheduling.Tolerations = append(c.Scheduling.Tolerations, hcp.Spec.Tolerations...)
```

Comment:
I was thinking if in terms of visibility it would be better to introduce a specific `c.setAdditionalTolerations(hcp)` and call it from `setLocation`.
Not 100% sure.

Reply:
makes sense. I changed this